src/wp-includes/fonts.php

```diff
@@ -122,6 +122,17 @@ function wp_get_font_dir() {
 		'error'   => false,
 	);
 
+	wp_mkdir_p( $defaults['path'] );
+
+	if ( ! wp_is_writable( $defaults['path'] ) ) {
+		$defaults            = wp_upload_dir();
```

Comment:
```suggestion
		$defaults            = wp_get_upload_dir();
```

This prevents the time based folder from being created, eg `uploads/20204/03`, if it doesn't already exist.

I think this will require https://github.com/WordPress/wordpress-develop/pull/6211 to prevent a loop. Are you able to add a test (there's one in the linked PR).

Reply:
added wp_get_upload_dir here https://github.com/WordPress/wordpress-develop/pull/6259/commits/895984bdf4c441d5369bb490e172c088077b3208